packages/react-menu/src/Menu.stories.tsx

```diff
@@ -114,54 +114,68 @@ export const MenuTriggerInteractions = () => {
   return <TextOnly openOnContext={context} openOnHover={hover} />;
 };
 
-export const NestedSubmenus = () => (
-  <Menu>
-    <MenuTrigger>
-      <button>Toggle menu</button>
-    </MenuTrigger>
+export const NestedSubmenus = () => {
+  const AppearanceSubMenu = () => (
```

Comment:
**q:** do we need to re-create the function on every render ? I'd advise to create another story file for those helpers if necessary

Reply:
until we have a better way of displaying the src code for the stories, it should stay. If this is our current docs, it should be clear to readers that we encourage separate components for sub menus